### PR TITLE
Return if we have no changed files

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

Return if we have no changed files

### CODE REVIEW

File: src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php [hunk 1]
```diff
@@ -843,17 +850,21 @@ protected function checkCategories($hookData, Github $github, Logger $logger, $p
 		// Get the files tha gets changed with this Pull Request
 		$files = $this->getChangedFilesByPullRequest($hookData, $github, $logger, $project);
 
-		// The new categories based on the current code of the PR
-		$newCategories = $this->checkFilesAndAssignCategory($files);
+		// If there are no changed files do nothing here.
+		if (!empty($files))
```
Comment (b2z) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: I would make here the same way.

if (empty($files))
{
        // If there are no changed files return
        return;
}

**Comment by zero-24** [comment]
should be like your suggestion now @b2z 😄

**Comment by elkuku** [comment]
Hey,
I'm currently trying to find out why the category assigning thingy is failing.
I somehow missed this PR, so I'd like to shoot jut one simple question here:

If I'm not wrong the code in this PR checks if a PR has files.
I don't believe that this is necessary.
So - Why?

**Comment by zero-24** [comment]
> I don't believe that this is necessary.

This is correct. But this is a test if that is the issue.

**Pull request merged**